src/SubMenu.js

```diff
@@ -1,42 +1,58 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 
+import AbstractMenu from './AbstractMenu';
 import { cssClasses, hasOwnProp } from './helpers';
 
-export default class SubMenu extends Component {
+export default class SubMenu extends AbstractMenu {
     static propTypes = {
         children: PropTypes.node.isRequired,
         title: PropTypes.node.isRequired,
         className: PropTypes.string,
         disabled: PropTypes.bool,
         hoverDelay: PropTypes.number,
-        rtl: PropTypes.bool
+        rtl: PropTypes.bool,
+        selected: PropTypes.bool,
+        onMouseMove: PropTypes.func.isRequired,
+        onMouseOut: PropTypes.func.isRequired,
+        forceOpen: PropTypes.bool.isRequired,
+        forceClose: PropTypes.func.isRequired,
+        parentKeyNavigationHandler: PropTypes.func.isRequired
```

Comment:
Ah sorry I didn't see the PropType warnings with only the usage of `npm start`. We can remove the isRequired or set defaultProps.
The properties are really required, but will be inject with `React.cloneElement(child, props)`, so before this the child has invalid props. In my opinion the best solution will be additional defaultProps. What do you guys think?